### PR TITLE
upd(brave-browser-nightly-deb): `1.37.44` -> `1.38.6`

### PR DESCRIPTION
**Pull request by theblobscp**

### CODE REVIEW

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Hashes don't match

**Comment by wizard-28** [comment]
@Korkmatik do you wish to continue maintaining this script?

**Comment by Korkmatik** [comment]
> @Korkmatik do you wish to continue maintaining this script?

Thanks for asking. Unfortunately currently I don't have time to maintain it.

File: packages/brave-browser-nightly-deb/brave-browser-nightly-deb.pacscript [hunk 1]
```diff
@@ -1,8 +1,8 @@
 name="brave-browser-nightly-deb"
-version="1.37.44"
+version="1.38.6"
 gives="brave-browser-nightly"
 breaks="${gives}-bin ${gives}-app"
 url="https://github.com/brave/brave-browser/releases/download/v${version}/${gives}_${version}_amd64.deb"
 description="The Brave browser is a fast, private and secure web browser for PC, Mac and mobile. This is the nightly build of Brave Browser"
-hash="4b656207fab91977a59a529a5ff2502c5e9f5969838174f11c4dbfeff46e37b8"
+hash="da93a1b5c7febb45686979927a4bc81e878bf144ee676c16e4e9d6f621e4a85f "
```
Comment (wizard-28) [inline packages/brave-browser-nightly-deb/brave-browser-nightly-deb.pacscript:7]: The hash is incorrect. The hash check fails in the CI.

File: packages/brave-browser-nightly-deb/brave-browser-nightly-deb.pacscript [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 maintainer="Korkmatik <9c54efc1-143b-4f5f-9d2f-f6f189583f0f@aleeas.com>"
```
Comment (wizard-28) [inline packages/brave-browser-nightly-deb/brave-browser-nightly-deb.pacscript:8]: Replace Korkmatik's details with yours @theblobscp . If you're willing to maintain it of course.

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Apparently the hashes still don't match

File: packages/brave-browser-nightly-deb/brave-browser-nightly-deb.pacscript [hunk 3]
```diff
@@ -1,8 +1,8 @@
 name="brave-browser-nightly-deb"
-version="1.37.44"
+version="1.38.6"
 gives="brave-browser-nightly"
 breaks="${gives}-bin ${gives}-app"
 url="https://github.com/brave/brave-browser/releases/download/v${version}/${gives}_${version}_amd64.deb"
 description="The Brave browser is a fast, private and secure web browser for PC, Mac and mobile. This is the nightly build of Brave Browser"
-hash="4b656207fab91977a59a529a5ff2502c5e9f5969838174f11c4dbfeff46e37b8"
-maintainer="Korkmatik <9c54efc1-143b-4f5f-9d2f-f6f189583f0f@aleeas.com>"
+hash="da93a1b5c7febb45686979927a4bc81e878bf144ee676c16e4e9d6f621e4a85f"
```
Comment (Henryws) [inline packages/brave-browser-nightly-deb/brave-browser-nightly-deb.pacscript:7]:
```suggestion
hash="002bc7729a53c69acbd2c1f52caa3e822d1fbc71a0c424b8f09be4b750ff3c74"
```

**Review by Henryws — CHANGES_REQUESTED** [review]
Proper hash

**Review by Henryws — APPROVED** [review]

File: packages/brave-browser-nightly-deb/brave-browser-nightly-deb.pacscript [hunk 4]
```diff
@@ -1,8 +1,8 @@
 name="brave-browser-nightly-deb"
-version="1.37.44"
+version="1.38.6"
 gives="brave-browser-nightly"
 breaks="${gives}-bin ${gives}-app"
```
Comment (wizard-28) [inline packages/brave-browser-nightly-deb/brave-browser-nightly-deb.pacscript:4]:
```suggestion
breaks="${gives}-bin ${gives}-app ${gives}-git ${gives}"
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Comment by theblobscp** [comment]
I'm not going to be able to mantain this, maybe some other time I might come back? But for now, this is 'stale'

**Comment by Henryws** [comment]
Hmmm, I might take up maintaining this as I currently use brave.

**Pull request closed without merging**